app/__init__.py

```diff
@@ -35,6 +43,11 @@ def create_app(config_name):
     return application
 
 
+@login_manager.user_loader
+def load_user(user_id):
```

Comment:
Ok, happy to go with this for now then, supplier user traffic will generally be pretty low so the API calls probably not too much of an issue.  We'll need some sort of solution by the last week of SSP though!